adb_shell/handle/usb_handle.py

```diff
@@ -161,7 +166,12 @@ class UsbHandle(BaseHandle):
     _HANDLE_CACHE = weakref.WeakValueDictionary()
     _HANDLE_CACHE_LOCK = threading.Lock()
 
-    def __init__(self, device, setting, usb_info=None, timeout_s=None):
+    def __init__(
+            self,
+            device: usb1.USBDevice,
```

Comment:
I prefer not to use type hints to preserve Python2 compatibility. I know Python2 is EOL’ed now, but unless it’s necessary I’d rather not break it. Instead, these types can be specified in the “Parameters” section of the docstring.

Reply:
I was actually planning on adding a PR to drop Python2 support. `:-(`
But OK, I'll respect that.

Reply:
Changed in https://github.com/Halastra/adb_shell/pull/5